Source/Fuse.Nodes/Tests/Fuse.Nodes.Test/Visual.Test.uno

```diff
@@ -9,8 +10,54 @@ using FuseTest;
 
 namespace Fuse.Test
```

Comment:
I've switched most test classe to the `FuseTest` namespace (without a dot) to distinguish them from being somehow part of `Fuse`.